content/docs/user-guide/concepts/data-pipelines.md

```diff
@@ -0,0 +1,36 @@
+---
+name: 'Data Pipelines'
+match: ['data pipeline', 'pipeline', 'pipelines']
+tooltip: >-
+  A [data pipeline](/doc/user-guide/concepts/data-pipelines) is a series of data
+  processing stages, chained by their outputs and inputs. They use some initial
+  data, may produce intermediate artifacts, and reach a final result.
+description: >-
+  In DVC, a data pipeline is a series of data processing stages. A pipeline may
+  produce intermediate data, and has a final result.
```

Comment:
- No need for "In DVC" I think
- Maybe use "They use some initial data, may produce intermediate artifacts, and reach a final result" (text I used in the tooltip)
- What key terms are we emphasizing?
- Has there been a SEO research on existing terms (per the GSoD project proposal)? E.g. I suspect "workfow" could be a good term to incorporate here (not just in the SEO desc but in the content).

Reply:
Updated the description with your suggestions. 

The existing terms to focus on are listed in the HTML comment, and I've added some more around "workflow" that I verified in search console (good idea).

Terms in the description have no direct impact on ranking, but the clickthrough ratio does. So for the description we should be doing our best to improve clickthrough (looking at terms with most potential). Based on DVC traffic, it's for sure "data pipelines", and the ML, data science, *ops terms listed are speculative based on related terms and searches.

Terms in the content do have a direct impact on the ranking, so that's where working in the keywords is most worthwhile.

Reply:
> So for the description we should be doing our best to improve clickthrough (looking at terms with most potential). 

An idea for how to improve CTR: Since this is a concept the first sentence of the description is good (definition of a pipeline). For the second sentence (~100 characters), we could use one or two key terms to describe uses for pipelines in DVC. Something like "Use DVC to build a machine learning pipeline or data science workflow."

> Terms in the content do have a direct impact on the ranking, so that's where working in the keywords is most worthwhile.

To further highlight the difference in approach: "workflow" comes up in a few different cases, so to improve rank beyond adding keywords, we might adjust or create an example to describe "building a machine learning (data science) workflow using pipelines in DVC". The better we can answer the query the more likely it will be rewarded. (Keeping in mind that we have to choose which queries are worth the effort.)

Reply:
Awesome, thanks for the tips!